project/Game.py

```diff
@@ -137,7 +153,15 @@ def quit_question(self, team, password):
         return True
 
     def answer_question(self, team, answer):
-        return True
+        if self.team.currentLandmark.answer is not answer:
+            self.team.add_penalty(1)
```

Comment:
Since we're inside the class couldn't we just directly set the penalty member instead of making a method call? self.penalty += 1